src/Components/WebView/test/E2ETest/BasicBlazorHybridTest.cs

```diff
@@ -18,21 +18,21 @@ public void Run()
         // Note: This test produces *a lot* of debug output to aid when debugging failures. The only output
         // that is necessary for the functioning of this test is the "Test passed" at the end of this method.
 
-        Console.WriteLine($"Current directory: {Environment.CurrentDirectory}");
-        Console.WriteLine($"Current assembly: {typeof(Program).Assembly.Location}");
+        Console.WriteLine($"[{DateTime.Now.ToLongTimeString()}] Current directory: {Environment.CurrentDirectory}");
```

Comment:
All these DateTime.Now's are so that each *line* within the log has the actual time it was generated. Otherwise all the lines get written out to the xUnit log at the same time and thus all have the same timestamp, which isn't useful if you're trying to see how long it took between lines.